### PR TITLE
add route for multiple network interface

### PR DESCRIPTION
**Pull request by bxy4543**
add route for multiple network interface

### CODE REVIEW

File: pkg/runtime/nodes.go [hunk 1]
```diff
@@ -123,6 +133,10 @@ func (k *KubeadmRuntime) deleteNodes(nodes []string) error {
 			if err := k.deleteNode(node); err != nil {
 				return fmt.Errorf("delete node %s failed %v", node, err)
 			}
+			err := k.deleteVIPRoute(node)
```
Comment (fanux) [inline pkg/runtime/nodes.go:136]: Why delete route here?

Comment (bxy4543) [reply pkg/runtime/nodes.go:136]: > Why delete route here?

after deleting a node, clear the traces of route addition.

File: pkg/runtime/nodes.go [hunk 2]
```diff
@@ -38,6 +38,12 @@ const (
 	RemoteAddRoute                  = "seautil route add --host %s --gateway %s"
 	RouteOK                         = "ok"
 	LvscareStaticPodCmd             = `echo "%s" > %s`
+	CheckNetworkIsDefaultRoute      = "ip route show |grep default |grep %s"
+	DeleteRouteCmd                  = "if ip route |grep %s;then ip route del %s;fi"
+	AddRouteCmd                     = "ip route |grep %s || ip route add %s"
+	DeleteRouteFromFile             = "if [ -f /etc/sysconfig/network-scripts/route-%s ]; then sed -i \"/%s/d\" /etc/sysconfig/network-scripts/route-%s;fi"
+	AddStaticRouteFile              = "cat /etc/sysconfig/network-scripts/route-%s|grep %s || echo %s >> /etc/sysconfig/network-scripts/route-%s"
+	GetNetworkInterface             = "ip route |grep -v %s |grep %s| awk -F '[ \\t*]' '{print $3}'"
```
Comment (fanux) [inline pkg/runtime/nodes.go:46]: Should use ip addr command to get it. grep node ip

Comment (kakaZhou719) [reply pkg/runtime/nodes.go:46]: Some hosts do not have `ip` command, use "net.Interfaces()" to get network interface

Comment (kakaZhou719) [reply pkg/runtime/nodes.go:46]: Some hosts do not have `ip` command. 
if remote: "ifconfig" cmd is system build-in. 
if local : use "net.Interfaces()".

Comment (fanux) [reply pkg/runtime/nodes.go:46]: Or use sealutil to check interface and add router.

**Review by fanux — APPROVED** [review]

**Pull request merged**